guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -18,14 +18,84 @@ You can list the available ece-tools commands using:
 php ./vendor/bin/ece-tools list
 ```
 
-The following updates describe the latest improvements to the ece-tools package, which update as needed through patching and product upgrades managed by the `magento-cloud-metapackage`. The ece-tools package adheres to the version sequence:  `200<major>.<minor>.<patch>`.
+The following updates describe the latest improvements to the `ece-tools` package, which uses the following version sequence:  `200<major>.<minor>.<patch>`.  See [Upgrades and patches]({{ site.baseurl }}/guides/v2.1/cloud/project/project-upgrade-parent.html) for information about updating to the latest release of the `ece-tools` package.
+
+
+## v2002.0.13
+
+
+#### New in the `ece-tools` package
+
+-  JIRA--MAGECLOUD-2169-->**Enable zero-downtime deployment**—Now you can configure {{site.data.var.ece}} to queue requests with required database changes during deployment and apply the changes as soon as the deployment completes. Requests can be held for up to 5 minutes to ensure that no sessions are lost. This feature is available when you configure the following Static Content Deployment (SCD) settings in the `magento.env.yaml` configuration file.
+
+   -  [Enable static content deployment on the build phase]({{ page.baseurl }}/cloud/env/variables-build.html#skip_scd):  `SKIP_SCD=false`
+   -  [Enable on-demand static content generation]({{ page.baseurl }}/cloud/env/variables-intro.html#scd_on_demand): `SCD_ON_DEMAND=true`
+
+-  **Docker Compose for Cloud**—Improved the [Docker configuration]({{ page.baseurl }}/cloud/reference/docker-config.html) process to launch a {{site.data.var.ece}} development environment in your local workspace. (Requires `ece-tools` version 2002.0.13 or later).
```

Comment:
We missed a lot here (assume that's because functionality was developed in sub-tasks). For Docker environments there were multiple improvements we do not cover here (please ask @shiftedreality review and help with it):
- MAGECLOUD-2360 (Added SSL Support)
- MAGECLOUD-2358 (Added Varnish image, which is deployed automatically but can be configured manually to work with your Docker installation)
- MAGECLOUD-2442 (Added Redis image, which is deployed & configured automatically to work with your Docker installation)
- MAGECLOUD-2361 (this is not an improvement bust still needs to be documented, since people were having issues with it)